CHANGELOG.md

```diff
@@ -45,10 +45,11 @@ More information about these warnings is available here: https://github.com/micr
 - Updated sample projects with latest changes.
 - Overloaded `AddDefaultDocumentConnectorProvider` and `AddDefaultDocumentContentExtractor` methods with a parameter to pass a function to calculate the length of a text and inject it as a dependency.
 - Added Readme file to all solution's projects.
-- Added event handler for `MemoryManager` operations.
+- Added event handler for `IMemoryStore` operations.
 - Added new extension method `GetKernelPromptAsync` in `Encamina.Enmarcha.SemanticKernel.Extensions.KernelExtensions` to retrieve the final prompt for a given prompt using the arguments.
 - Added new extension method `GetKernelFunctionUsedTokensFromPromptAsync` in `Encamina.Enmarcha.SemanticKernel.Extensions.KernelExtensions` to obtain the total number of tokens used in generating a prompt from an inline prompt function.
 - Fixed `GetMaxTokensFromKernelFunction` in `Encamina.Enmarcha.SemanticKernel.Extensions.KernelExtensions`. Now, it considers whether the arguments are of type `OpenAIPromptExecutionSettings` when obtaining the MaxTokens.
+- Added `IMemoryStoreExtender` class. This class convert to obsolete the `IMemoryManager` class and their references.
```

Comment:
In English, adverbs to participles (which is common in Spanish) are hard to read. It would be better something like:
"_New `IMemoryStoreExtender` type which obsoleces the `IMemoryManager` type and its references._"

Reply:
Thank you for your contribution. I will update it.